forc/src/cli/commands/predicate_root.rs

```diff
@@ -18,6 +18,10 @@ pub struct Command {
     pub build_output: BuildOutput,
     #[clap(flatten)]
     pub build_profile: BuildProfile,
+
+    #[clap(long)]
+    /// Experimental flags for the "new encoding" feature
```

Comment:
there are a few of these, feel free to ignore if you want.

Reply:
Fixed.